app/src/db/cbt-db.ts

```
@@ -313,43 +321,57 @@ class CBTDatabase extends Dexie {
+    testOutputDatas = viaJson ? utilCloneJson(testOutputDatas) : testOutputDatas
+    const testNotes: (TestNotes | null)[] = []
 
     for (const testOutputData of testOutputDatas) {
       testOutputData.testResultOverview = utilGetTestResultOverview(testOutputData)
 
-      if (checkAndSkipExisting) {
-        const existing = await this.getTestResultOverviewByCompoundIndex(testOutputData)
-        if (existing) continue
+      if (testOutputData.testNotes) {
+        testNotes.push(testOutputData.testNotes)
+        delete testOutputData.testNotes
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)